rosidl_generator_py/test/test_interfaces.py

```diff
@@ -24,8 +24,8 @@
 def test_strings():
     a = Strings()
 
-    assert(a.empty_string is str())
-    assert(a.def_string == 'Hello world!')
+    assert a.empty_string is str()
```

Comment:
I think the goal here was to confirm that it was the language default initialization value which is more restrictive that checking the type. Would it be less confusing to use `assert a.empty_string is ''` ?